index.html

```diff
@@ -49,173 +53,172 @@ <h1 class="site-header__title">
       </a>
     </header>
 
-    <main class="content">
-
-      <div class="projects">
-
-        <div class="project">
-          <div class="project__header project__header--dispatch">
-            <svg class="project__icon"><use xlink:href="#icon--ama" /></svg>
-            <h2 class="project__title">Volunteer Dispatch for Astoria Mutual Aid Network</h2>
-          </div>
-          <p class="project__description">The system that Astoria Mutual Aid Network uses to connect requests for help from neighbors with the network’s volunteers. Astoria Mutual Aid Network came about at the beginning of the pandemic, and our team was created to support their work.</p>
-          <div class="project__links">
-            <a class="project__link" href="https://astoriamutualaid.com/">
-              <svg class="project__link-icon"><use xlink:href="#icon--link" /></svg>
-              <span class="project__text">Astoria Mutual Aid Network</span>
-            </a>
-            <a class="project__link" href="https://github.com/astoria-tech/volunteer-dispatch">
-              <svg class="project__link-icon"><use xlink:href="#icon--filecode" /></svg>
-              <span class="project__text">View Code</span>
-            </a>
+    <main>
```

Comment:
The `<main>` tag should wrap all essential page content to be semantically correct, so here that would include the proceeding `<section>` and `<div  class="aboutus-columns">` tag.

So the top level tags would look like:

```
<header class="site-header">...</header>
<main>...</main>
<footer  class="site-footer">...</footer>
```

Styles may need to be adjusted slightly to accommodate this nesting.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main